src/plugins/bfetch/public/plugin.ts

```diff
@@ -83,6 +84,7 @@ export class BfetchPublicPlugin
         headers: {
           'Content-Type': 'application/json',
           'kbn-version': version,
+          [X_ELASTIC_INTERNAL_ORIGIN_REQUEST]: 'Kibana',
```

Comment:
`bfetch` has it's own way of making browser-side HTTP calls and needs to adopt the header too.